wordpress-mobile/WordPress-Android
# Ube cannot view or interact with the classic block on jetpack sites

Gutenberg PR:  https://github.com/WordPress/gutenberg/pull/26168
Gutenberg mobile PR:  https://github.com/wordpress-mobile/gutenberg-mobile/pull/2730

To test:

1. On the web, create a post with a classic block on a Jetpack site (we should use jurassic ninja to test the classic block on a jetpack site).
2. Save or publish the post.
3. In the app, open the post from the previous step in the block editor.
4. Tap on the classic block.
5. Tap on the classic block again.
6. Select the option to "Edit using the web editor."
7. Observe that classic block is presented and all HTML elements (navigation bar and other) are hidden.
8. Check that WebView has a good block title (e.g. 'Edit Classic block')

PR submission checklist:
